src/trigger-circleci-workflows.ts

```diff
@@ -61,17 +73,58 @@ async function triggerCircleCI(
         );
       }
     });
-  context.log.info(
-    {data},
-    `Build triggered successfully for ${context.payload['pull_request']['html_url']}`
-  );
+  context.log.info({data}, `Build triggered successfully for ${repoKey}`);
 }
 
 export function circlePipelineEndpoint(repoKey: string): string {
   return `/api/v2/project/github/${repoKey}/pipeline`;
 }
 
-async function runBot(context: probot.Context): Promise<void> {
+export function genParametersForPR(
+  config: object,
+  context: probot.Context,
+  appliedLabels: string[]
+): object {
+  context.log.debug({config, appliedLabels}, 'genParametersForPR');
+  const parameters = {};
+  const labelsToParams = config['labels_to_circle_params'];
+  for (const label of Object.keys(labelsToParams)) {
+    // ci/all is a special label that will set all to true
+    if (appliedLabels.includes('ci/all') || appliedLabels.includes(label)) {
+      parameters[labelsToParams[label].parameter] = true;
```

Comment:
Wondering if we need a config for this at all: can't we just define some predictable munge (e.g., drop the leading `ci/`) and force parameters and labels to line up this way?

Reply:
Well, I guess the naming conventions before and after are pretty different, so whatever. (Also this was in the old PR, it seems.)